djstripe/models/webhooks.py

```diff
@@ -25,6 +25,32 @@ def _get_version():
     return __version__
 
 
+def get_remote_ip(_request):
+    """Given the HTTPRequest object return the IP Address of the client
+
+    :param _request: client request
+    :type _request: HTTPRequest
+
+    :Returns: the client ip address
+    """
+
+    # HTTP_X_FORWARDED_FOR is relevant for django running behind a proxy
+    x_forwarded_for = _request.META.get("HTTP_X_FORWARDED_FOR")
+    if x_forwarded_for:
+        ip = x_forwarded_for.split(",")[0]
```

Comment:
Please add a test for this case. The test can just directly test the function `get_remote_ip()`, it doesn't have to trigger the whole webhook pipeline.